src/Behat/Cores/Drupal8.php

```diff
@@ -67,6 +68,19 @@ public function runElysiaCronJob($job) {
     throw new PendingException('Elysia job cron run not implemented yet!');
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function runUltimateCron($cron_name) {
+    $cron_job = current(\Drupal::entityTypeManager()->getStorage('ultimate_cron_job')->loadByProperties(['id' => $job]));
```

Comment:
This method should throw an exception if Ultimate cron module is not installed.

Reply:
Updated method, thank you, please review it.